src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenCapturing.cs

```diff
@@ -422,7 +422,11 @@ private void SerializeMethod(MethodInfo methodInfo, StringBuilder builder, int m
         /// a maximum number of local functions, and a maximum scope depth to decide the
         /// limits of the combinations.
         /// </summary>
+#if TEST_IOPERATION_INTERFACE
```

Comment:
Are we skipping them because those tests take long time to finish (even without ioperation validation), or because it doesn't make sense to test them for ioperation? `ValidateIOperations` method would be a no-op already if `TEST_IOPERATION_INTERFACE` constant is defined, so I guess this might be unnecessary if execution time is not a concern.

Reply:
The test hook is either taking too long for skipped tests, or it affects test scenarios in an undesired way.  

> ValidateIOperations method would be a no-op already if TEST_IOPERATION_INTERFACE constant is defined

I don't think this is accurate.

Reply:
Why is it inaccurate? The entire method body would #if'd out if `TEST_IOPERATION_INTERFACE` is not defined.
https://github.com/dotnet/roslyn/blob/14aed138a01c448143b9acf0fe77a662e3dfe2f4/src/Test/Utilities/Portable/Compilation/CompilationExtensions.cs#L258

Reply:
> Why is it inaccurate? The entire method body would be #if'd out if TEST_IOPERATION_INTERFACE is not defined.

That is true. However, your original comment was about situation when  TEST_IOPERATION_INTERFACE constant **is defined**.